### PR TITLE
Lack of initialization in matrices and vectors

### PR DESCRIPTION
**Pull request by juliobarna**
Correcting lack of initialization in all the matrices  and vectors involve in the evaluation of the pressure.

### CODE REVIEW

File: applications/incompressible_fluid_application/custom_elements/fluid_3dGLS_expl.cpp [hunk 1]
```diff
@@ -938,8 +955,53 @@ void Fluid3DGLS_expl::GetDofList(DofsVectorType& ElementalDofList,ProcessInfo& C
...
+            viscous_part += DN_DX(i,k)*DN_DX(i,k);
+    viscous_part *= nu;
 
 
+    double conv_part = 2.0*norm_u/h;
+
+    const double dyn_st_beta = CurrentProcessInfo[DYNAMIC_TAU];
+    const double inv_dt_coeff = CurrentProcessInfo[BDF_COEFFICIENTS][0];
+    double tau = 1.00 / (dyn_st_beta * inv_dt_coeff + viscous_part + conv_part);
+
```
Comment (lorenzogracia) [inline applications/incompressible_fluid_application/custom_elements/fluid_3dGLS_expl.cpp:987]: Some lines are unnecessary

File: applications/incompressible_fluid_application/custom_elements/fluid_3dGLS_expl_comp.cpp [hunk 2]
```diff
@@ -1020,7 +1045,52 @@ void Fluid3DGLS_expl_comp::GetDofList(DofsVectorType& ElementalDofList,ProcessIn
...
+
+
+    double conv_part = 2.0*norm_u/h;
+
+    const double dyn_st_beta = CurrentProcessInfo[DYNAMIC_TAU];
+    const double inv_dt_coeff = CurrentProcessInfo[BDF_COEFFICIENTS][0];
+    double tau = 1.00 / (dyn_st_beta * inv_dt_coeff + viscous_part + conv_part);
+
+
+
```
Comment (lorenzogracia) [inline applications/incompressible_fluid_application/custom_elements/fluid_3dGLS_expl_comp.cpp:1082]: Same here

**Review by lorenzogracia — DISMISSED** [review]

**Comment by juliobarna** [comment]
@lorenzogracia

**Review by RiccardoRossi — APPROVED** [review]
apart for lorenzo's comment, i see nothing harmful.

i am approving, but please adrress Lorenzo's comment

**Comment by juliobarna** [comment]
@RiccardoRossi  thanks!

**Comment by philbucher** [comment]
Can this branch be deleted?

**Comment by playpaolo** [comment]
the branch can be deleted now

**Pull request merged**